kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/adapter/QuickConversationAdapter.java

```diff
@@ -255,8 +254,9 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
                 } else if (message.getContentType() == Message.ContentType.PRICE.getValue()) {
                     myholder.messageTextView.setText(EmoticonUtils.getSmiledText(context, ConversationUIService.FINAL_PRICE_TEXT + message.getMessage(), emojiconHandler));
                 } else if (message.getContentType() == Message.ContentType.TEXT_HTML.getValue()) {
-                    myholder.messageTextView.setText(Html.fromHtml(message.getMessage()));
-                    if (DetailedConversationAdapter.isEmailTypeMessage(message)) {
+//                    myholder.messageTextView.setText(Html.fromHtml(message.getMessage()));
```

Comment:
Can we remove this code instead of commenting it out ?

Reply:
Yes we can